### ISSUE TITLE
현재 라이브러리 install이 안되나요?

### ISSUE THREAD

**Issue by yoct1994** [jus0k/scroll-hooks#20]
이 라이브러리 이용해서 웹사이트를 만들고 싶습니다만 install칸에 yarn말고 없네요..

**Comment by jus0k** [comment]
> 이 라이브러리 이용해서 웹사이트를 만들고 싶습니다만 install칸에 yarn말고 없네요..

네 ㅜㅜ 아직 라이브러리화 까지는 안되어 있어요! 원본 코드 참고해주시면 금방 따라서 만드실 수 있을 것 같습니다! 
내년에는 라이브러리화해서 업데이트 한번 해보겠습니다!
